base/wheels/Dockerfile

```diff
@@ -14,18 +14,22 @@ RUN echo "GDAL==$(gdal-config --version)" > /tmp/constraints.txt \
   GDAL \
   && rm /tmp/constraints.txt
 
+COPY ./constraints.txt constraints.txt
+
 RUN pip3 wheel \
   --no-cache-dir \
   --no-binary=:all:  \
   --no-build-isolation \
   --no-deps \
+  -c /wheels/constraints.txt \
```

Comment:
I think it's better to spell out the full `--flag` in CI and scripts in general, so that it's more readable and super clear.